index.js

```diff
@@ -1,55 +1,94 @@
 const WebSocket = require('ws');
 const request = require('request');
 const getParser = require('@streammedev/parse-message');
+const program = require('commander');
```

Comment:
ideally you break this out into a script

/bin/streammebot
/src/index.js

and bin/streammebot is an executable that also goes in your package.json